egs3/TEMPLATE/tts/src/hf_demo_readme.md

```
@@ -0,0 +1,37 @@
+${description}
+
+## Demo
+
+- Space: `${hf_repo}`
+- Model: `${model_ref}`
+- Creator: `${creator}`
+
+
+## Citation
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)